Marketplace/controllers/api/apiUserController.js

```diff
@@ -1,11 +1,49 @@
-const db = require("../../database/models");
-const entidad = db.Usuario;
+const usuarioRepository = require("../../repositories/usuarioRepository");
 
 module.exports = {
-	listado: (req, res) => {
-
-	},
-	detalle: (req, res) => {
-		
-	},
+	listado: async (req, res) => {
+        let usuarios = await usuarioRepository.ObtenerTodos();
+        let usuarioRespuesta = usuarios.map(x => {
+            return {
+                id: x.id,
+                nombre: x.nombre,
+				email: x.email
+            }
+        });
+        let respuesta = {
+            meta: {
+				   TotalUsuarios: usuarios.length,
+				   url: '/api/usuarios'
+			    },
+            data: usuarioRespuesta
+        };
+        res.status(200).json(respuesta);
```

Comment:
En este endpoint falta la url de como conseguir el detalle del usuario

Reply:
diego tmb me hizo este comentario le agrego la url de detalle de cada usuario.

Reply:
Cris estuve investigando en la web y la gran mayoria aconseja hacer una vista personalizada para el error 404 (la vista no la tenemos), porque la standar no arroja grandes detalles(no la pude hacer funcionar). La unica que pude hacer funcionar es res.send(404), pero arroja un "not found" pequeño en la pagina y nose si queda tan bien. Lo dejo como estaba y de acuerdo a lo que me conteste pablo vemos que hacemos. Por el momento subo los cambio de las otras cosas que se pidieron.

Reply:
claro, la vista 404 es algo del front end, dejalo como 200 👍